### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by RedFeuer** [community/community#132790] [GitHub Education] [answered]
### Hi everyone,

my request was submitted on July 16
when will I be able to use the Student developer pack?
![chrome_GchwsXQC4V](https://github.com/user-attachments/assets/171e7df8-57af-4f8d-96fc-098b2a54e4d8)

thank You!

**Comment by davevad93** [comment, marked as answer]
Hi @RedFeuer ,

It usually takes a few days, but sometimes it can even take a couple of weeks; the waiting time may vary depending on different factors, e.g. the volume of applications.

When the benefits become available, the status label will turn purple and change to:

`Congrats! Your request for @RedFeuer was approved and your benefits are now available.`

**Reply by RedFeuer** [reply]
Got student developer pack.
Thank You!

**Reply by davevad93** [reply]
You're welcome @RedFeuer !

**Reply by StanislavUhimenko** [reply]
Молодець все правильно але треба опублікувати 
[](url)
